Resources/Computing/ComputingElement.py

```diff
@@ -78,7 +78,7 @@ def __init__(self, ceName):
     self.mandatoryParameters = []
     self.batch = None
     self.batchSystem = None
-    self.batchModuleFile = None
+    self.parallelLibrary = None
```

Comment:
```suggestion
    self.parallelLibrary = None
    self.parallelLib = None
```

Reply:
- I removed `self.batchSystem` (the name of the batch system) and `self.parallelLibrary` (the name of the parallel library) attributes from `ComputingElement.py`.
- I renamed `self.batch` as `self.batchSystem`

I did not replace `self.parallelLibrary` because it was not really useful.
I added a `BatchSystem` class with a `name` attribute to replace the former `self.batchSystem` attribute.